pages/meet/index.tsx

```diff
@@ -8,6 +9,7 @@ function Meet({ }: Props): ReactElement {
 	return (
 		<div>
 			Meet
+			<Quote />
```

Comment:
This component will ultimately live on the signup page, I'm glad you left it here so I could view it but you can pull it from here for now until the signup component is merged in and then we can combine the two.